### ISSUE TITLE
prtvol2csv, support same YML-file as Webviz

### ISSUE THREAD

**Issue by berland** [equinor/subscript#330]
Webviz already defines a yaml-syntax that prtvol2csv could have used as input to its FIPNUM <-> (Region, Zone) mapping.

An example is:

```yaml
FIPNUM:
  groups:
    REGION:
      WestLowland: [1, 8, 15]
      CentralSouth: [2, 9, 16]
      CentralNorth: [3, 10, 17]
      NorthHorst: [4, 11, 18]
      CentralRamp: [5, 12, 19]
      CentralHorst: [6, 13, 20]
      EastLowland: [7, 14, 21]
    ZONE:
      Valysar: [1, 2, 3, 4, 5, 6, 7]
      Therys: [8, 9, 10, 11, 12, 13, 14]
      Volon: [15, 16, 17, 18, 19, 20, 21]
```

Suggest to add support for this yaml format to `prtvol2csv` to allow using the same input file for both Webviz and prtvol2csv.

**Comment by berland** [comment]
Webviz' syntax is described by https://github.com/equinor/webviz-subsurface/blob/7e4a1c76971b4f4e42559ac61618890c46645f72/webviz_subsurface/plugins/_reservoir_simulation_timeseries_regional.py#L1422

**Comment by berland** [comment]
@asnyv and  @HansKallekleiv, I suppose this part of the webviz-subsurface API is stable?

**Comment by asnyv** [comment]
Sort of @berland, but I regret adding the "groups" level. Have been thinking many times of why I added it, but cannot come up with a reasonable explanation.

**Comment by asnyv** [comment]
Think we should have jumped straight to "REGION", "ZONE" and so on after FIPNUM?

**Comment by berland** [comment]
If you plan to remove `groups`, `prtvol2csv` can easily support both. I don't either know why `groups` is there..

**Comment by asnyv** [comment]
Would surprise me if you did @berland. I am the one to blame here. Think it is most likely because there was another format tested first where it was used for _something_, and in a weak moment it wasn't removed in the final one. 

Main reason it hasn't been changed is that it works, hasn't been looked much at since, and haven't been too keen on breakkng it. 

But a unified format would be nice. (Just need to support any FIPXXX with any subgroups, like REGION, ZONE and/or WHATEVER).

Without the _groups_, I think this one is fairly ok

**Comment by berland** [comment]
The PR now includes support for a possible future webviz format  (with `groups` removed).

**Comment by berland** [comment]
Fixed in #331
